miso-web/src/main/webapp/pages/editLibrary.jsp

```diff
@@ -568,124 +568,12 @@
-              <td id="tarSeq${dil.id}">
-                <c:if test="${empty dil.targetedSequencing}">
-                  NONE
-                </c:if>
-                <c:if test="${not empty dil.targetedSequencing}">
-                  ${dil.targetedSequencing.alias}
-                </c:if>
-              </td>
-            </c:if>
-            <td class="fit" id="idBarcode${dil.id}">
-              <c:if test="${not empty dil.identificationBarcode}">
-                <div class="barcodes">
-                  <div class="barcodeArea ui-corner-all">
-                    <ul class="barcode-ddm">
-                      <li>
-                        <a onmouseover="mopen('dil${dil.id}IdBarcodeMenu')" onmouseout="mclosetime()">
-                          <span style="float:right; margin-top:6px;" class="ui-icon ui-icon-triangle-1-s"></span>
-                          <span id="dil${dil.id}IdBarcode" style="float:right"></span>
-                        </a>
-
-                        <div id="dil${dil.id}IdBarcodeMenu" onmouseover="mcancelclosetime()" onmouseout="mclosetime()">
-                          <a href="javascript:void(0);" onclick="Library.barcode.printDilutionBarcode(${dil.id}, '${library.platformType}');">Print</a>
-                        </div>
-                      </li>
-                    </ul>
-
-                    <script type="text/javascript">
-                      jQuery(document).ready(function () {
-                        Fluxion.doAjax(
-                          'libraryControllerHelperService',
-                          'getLibraryDilutionBarcode',
-                          {'dilutionId':${dil.id},
-                            'url': ajaxurl
-                          },
-                          {'doOnSuccess': function (json) {
-                            jQuery('#dil${dil.id}IdBarcode').html(
-                              "<img style='border:0;' src='<c:url value='/temp/'/>" + json.img + "'/><br/>"
-                            );
-                          }
-                          });
-                      });
-                    </script>
-                  </div>
-                </div>
-
-              </c:if>
-            </td>
-            <td id="edit${dil.id}" align="center">
-              <a href="javascript:void(0);" onclick="Library.dilution.changeLibraryDilutionRow('${dil.id}',${autoGenerateIdBarcodes}, ${detailedSample})">
-                <span class="ui-icon ui-icon-pencil"></span>
-              </a>
-            </td>
-            <td>
-              <a href="<c:url value="/miso/poolwizard/new/${library.sample.project.id}"/>">Construct New Pool</a>
-            </td>
-          </tr>
-        </c:forEach>
-      </c:if>
-      </tbody>
-    </table>
-    <input type='hidden' id='dilLibraryId' name='id' value='${library.id}'/>
-  </form>
-</div>
-
+  <miso:list-section name="Dilutions" target="dilution" items="${libraryDilutions}" config="${libraryDilutionsConfig}"/>
```

Comment:
The `Pool together` option doesn't make sense in the context of this page -- dilutions derived from the same library would have the same indices, so no way to demultiplex. Worth putting in the effor to remove that possibility for error?